src/mlpack/methods/decision_tree/log_loss.hpp

```diff
@@ -0,0 +1,128 @@
+/**
+ * @file methods/decision_tree/gain_functions/log_loss.hpp
+ * @author Abhimanyu Dayal
+ *
+ * The logistic gain class, which is a gain function for xgboost.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_XGBOOST_LOG_LOSS_HPP
+#define MLPACK_METHODS_XGBOOST_LOG_LOSS_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <cmath>
```

Comment:
Trivial comment but I think that this is already included by `prereqs.hpp` or `base.hpp`. :+1: